tox.ini

```diff
@@ -0,0 +1,47 @@
+[tox]
+envlist =
+    clean,
+    check,
+    py,
+    report,
+    cli,
+
+[testenv]
+deps =
+    pytest
+    pytest-cov
+depends =
+    py: clean
+    report: py
+commands =
+    #pytest tests {posargs}
+    pytest --cov=relaycard --cov-report=term-missing tests {posargs}
+
+[testenv:check]
+deps =
+    black
+    mypy
+    ruff
+skip_install = true
+commands =
+    python setup.py check --strict --metadata
+    ruff check .
+    black --check --diff .
+    mypy --install-types --non-interactive .
```

Comment:
Does it make sense to have mypy running if we don't do type annotations? Or do you intent to add them later?

Reply:
Done